frontend/src/lib/components/HTMLElementsDisplay/htmlElementsDisplayLogic.ts

```diff
@@ -111,6 +118,18 @@ export const htmlElementsDisplayLogic = kea<htmlElementsDisplayLogicType>([
                     : 'warning'
             },
         ],
+        hasHiddenElements: [
+            (s) => [s.elements, s.visibleElements],
+            (elements, visibleElements) => {
+                return elements.length > visibleElements
+            },
+        ],
```

Comment:
I know this selector isn't really necessary, considering `visibleElements`. But it makes the code a bit more readable.